### PR TITLE
Fix visualping

### PR DESCRIPTION
**Pull request by luancazarine**
### <samp>🤖 Generated by Copilot at edd9ab7</samp>

Added two new components and updated an existing one. The new components are `handwrytten` and `signaturit`, which enable sending handwritten notes and signing documents, respectively. The updated component is `visualping-create-job`, which fixes a typo and bumps the version.

### <samp>🤖 Generated by Copilot at edd9ab7</samp>

> _We create and we destroy, we are the masters of the code_
> _We fix the typos and the bugs, we make the actions run and load_
> _We add new components to the mix, we lock the dependencies down_
> _We improve the functionality, we make the repository sound_

### <samp>🤖 Generated by Copilot at edd9ab7</samp>

*  Renamed `ttargetDevice` to `targetDevice` in `visualping-create-job` action to fix typo and improve consistency ([link](https://github.com/PipedreamHQ/pipedream/pull/6496/files?diff=unified&w=0#diff-a0f1a6fb587ac3da3e3e62b9fba814613df0ca566622ef587f8f6a1f4aca0f55L100-R100))
* Updated version of `visualping-create-job` action from `0.0.1` to `0.0.2` to reflect changes ([link](https://github.com/PipedreamHQ/pipedream/pull/6496/files?diff=unified&w=0#diff-a0f1a6fb587ac3da3e3e62b9fba814613df0ca566622ef587f8f6a1f4aca0f55L6-R6))
* Added `components/handwrytten` and `components/signaturit` entries to `pnpm-lock.yaml` file to lock dependencies of new components ([link](https://github.com/PipedreamHQ/pipedream/pull/6496/files?diff=unified&w=0#diff-32824c984905bb02bc7ffcef96a77addd1f1602cff71a11fbbfdd7f53ee026bbR1860-R1862), [link](https://github.com/PipedreamHQ/pipedream/pull/6496/files?diff=unified&w=0#diff-32824c984905bb02bc7ffcef96a77addd1f1602cff71a11fbbfdd7f53ee026bbR3970-R3972))

### CODE REVIEW

File: components/visualping/actions/create-job/create-job.mjs [hunk 1]
```diff
@@ -3,7 +3,7 @@ import visualping from "../../app/visualping.app.mjs";
 export default {
   key: "visualping-create-job",
   name: "Create A New Job",
-  version: "0.0.1",
+  version: "0.0.2",
```
Comment (dylburger) [inline components/visualping/actions/create-job/create-job.mjs:6]:
```suggestion
  version: "0.0.2"
```

File: pnpm-lock.yaml [hunk 2]
```diff
@@ -1857,6 +1857,9 @@ importers:
     dependencies:
       '@pipedream/platform': 1.5.1
 
+  components/handwrytten:
```
Comment (dylburger) [inline pnpm-lock.yaml:1860]:
```suggestion
  components/handwritten:
```

File: pnpm-lock.yaml [hunk 3]
```diff
@@ -1857,6 +1857,9 @@ importers:
...
+    specifiers: {}
```
[the remark this answers is not in the document]
Comment (dylburger) [reply pnpm-lock.yaml:1861]: No changes needed

File: pnpm-lock.yaml [hunk 4]
```diff
@@ -3964,6 +3967,9 @@ importers:
     dependencies:
       '@pipedream/platform': 0.10.0
 
+  components/signaturit:
```
Comment (dylburger) [inline pnpm-lock.yaml:3970]:
```suggestion
  components/signature:
```

Comment (dylburger) [reply pnpm-lock.yaml:3970]: Changed "signaturit" to "signature" to correct spelling

File: pnpm-lock.yaml [hunk 5]
```diff
@@ -3964,6 +3967,9 @@ importers:
...
+    specifiers: {}
```
Comment (dylburger) [inline pnpm-lock.yaml:3971]:
```suggestion
    specifiers: {}
```

**Review by andrewjschuang — APPROVED** [review]

**Pull request merged**